src/event.rs

```diff
@@ -858,6 +861,7 @@ where
 						channel_id,
 						user_channel_id: UserChannelId(user_channel_id),
 						counterparty_node_id,
+						reason: Some(reason),
```

Comment:
A few lines up we log `reason` with `Debug`. Should we use `Display` given it is implemented for `ClosureReason`?